src/http/server.rs

```diff
@@ -56,8 +71,12 @@ fn create_allowed_headers(headers: &HeaderMap, allowed: &HashSet<String>) -> Hea
 
   new_headers
 }
+
 pub async fn start_server(file_path: &String) -> Result<()> {
   let server_sdl = fs::read_to_string(file_path)?;
+    
+  validate_schema(&server_sdl).context("Schema validation failed")?;
```

Comment:
Validation needs to be implemented inside of `from_config.rs`

Reply:
aaded checks in from_config.rs